sdk/storage/azure-storage-blob/tests/perfstress_tests/T1_legacy_tests/list_blobs.py

```diff
@@ -0,0 +1,31 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.perfstress_tests import get_random_bytes
+
+from ._test_base_legacy import _LegacyContainerTest
+
+
+class LegacyListBlobsTest(_LegacyContainerTest):
+
+    async def global_setup(self):
+        await super().global_setup()
+        for i in range(self.args.num_blobs):
```

Comment:
It should be faster to create many blobs in parallel using a reasonable number of threads (like 32).  Here's a helper function we wrote in JS for this purpose, Python should be able to use similar (with threads instead of async tasks):

https://github.com/mikeharder/blob-upload-parallel/pull/2/files#diff-0634662c6970258c08f765262734d321f014ad9a55ce51c0b9519b34fcbd6d7aR26-R49

Reply:
Especially since this is Track 1, and we use async for the Track 2 version of the test, I'm fine not changing this.